pages/developers/gha_wheels.md

```diff
@@ -103,7 +103,7 @@ The core of the work is down here:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, windows-2019, macos-10.15]
+        os: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
@henryiii has made the comment that this exposes us every now and then to issues with fresh versions - has happened. Dunno, really.

Reply:
fair enough, makes sense and I am sure @henryiii knows that well. changed that back